### PR TITLE
Episode 2: First draft about documentation section

### PR DESCRIPTION
**Pull request by bebatut**

### CODE REVIEW

File: _episodes/02-make-it-public.md [hunk 1]
```diff
@@ -25,14 +25,85 @@ keypoints:
...
 # Documentation
 
-* GH guide to GH Pages
-* RTD guide
+To make people using or contributing, a software needs to be documented.
+
+> ## Discussion
+>
+> - What experiences have you had with good or bad documentation?
+> - What was important for you **as an user**?
```
Comment (mkuzak) [inline _episodes/02-make-it-public.md]: as a user

File: _episodes/02-make-it-public.md [hunk 2]
```diff
@@ -25,14 +25,85 @@ keypoints:
...
+> 1. What information was missing?
+> 2. Talk to your neighbour and see if your lists agree.
+>
+> > ## Solution
+> > - No input definition
+> > - 
+> {:.solution}
+{:.challenge}
+
+One of the main challenge of documentation is to keep it up-to-date, following the evolution of the software. To avoid this issue, a good documentation follow these recommendation:
```
Comment (mkuzak) [inline _episodes/02-make-it-public.md]: these recommendations

File: _episodes/02-make-it-public.md [hunk 3]
```diff
@@ -25,14 +25,85 @@ keypoints:
...
+> > - Go back to the root repository on GitHub
+> > - Click on the `Edit` button
+> > - Paste the link to `Website`
+> > - Click on `Save`
+> >
+> {:.solution}
+>
+{:.challenge}
+
+More complex documentation can be also be generated using [Sphinx](http://www.sphinx-doc.org/en/master/). For example, the docstrings for each functions could be extracted to populate an extensive API documentation. Even complex, this documentation can be stored on GitHub: it is only text.
```
Comment (mkuzak) [inline _episodes/02-make-it-public.md]: for each function could

**Review by mkuzak — CHANGES_REQUESTED** [review]

File: _episodes/02-make-it-public.md (hunk 1 above)
Comment (tobyhodges) [reply _episodes/02-make-it-public.md]: Ugh... I apologise on behalf of the English language.

Comment (bebatut) [reply _episodes/02-make-it-public.md]: 😄

**Comment by tobyhodges** [comment]
Thank you, Bérénice

**Pull request merged**